StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodAllowRedisplay.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  STPPaymentMethodAllowRedisplay.swift
+//  StripePayments
+//
+
+import Foundation
+
+/// Values for STPPaymentMethodAllowRedisplay
+@objc public enum STPPaymentMethodAllowRedisplay: Int {
+    /// This is the default value for payment methods where allow_redisplay wasn’t set.
+    case unspecified
+
+    /// Use limited to indicate that this payment method can’t always be shown to a customer in a checkout flow. For example, it can only be shown in the context of a specific subscription.
+    case limited
+
+    /// Use always to indicate that this payment method can always be shown to a customer in a checkout flow.
+    case always
+
+    var stringValue: String? {
+        switch self {
+        case .unspecified:
+            return nil
```

Comment:
Should this be "unspecified"?

Reply:
Not specifying it will result in 'unspecified', but true..  I think it makes sense to just send 'unspecified' instead of having a nil value.